tests/src/e2e/helpers.rs

```diff
@@ -365,3 +365,11 @@ pub fn epoch_sleep(
         }
     }
 }
+
+/// Wait for txs and VPs WASM compilations to finish. This is useful to avoid a
+/// timeout when submitting a first tx.
+pub fn wait_for_wasm_pre_compile(ledger: &mut NamadaCmd) -> Result<()> {
+    ledger.exp_string("Finished compiling all")?;
+    ledger.exp_string("Finished compiling all")?;
```

Comment:
why 2?

Reply:
there are instances of wasm cache - vp and tx

Reply:
ah right right. perfect then!